packages/appstore/src/components/demo-reset-balance/demo-reset-balance.tsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import { Icon, Text, Button, Div100vhContainer } from '@deriv/components';
+import { Localize } from '@deriv/translations';
+import { useRequest } from '@deriv/api';
+import { observer, useStore } from '@deriv/stores';
+import './demo-reset-balance.scss';
+
+type TDemoResetBalanceProps = {
+    setActiveTabIndex: (index: number) => void;
+};
+
+const DemoResetBalance = observer(({ setActiveTabIndex }: TDemoResetBalanceProps) => {
+    const { mutate, isSuccess: isResetBalanceSuccess } = useRequest('topup_virtual');
+    const { client, ui } = useStore();
+    const { accounts, loginid } = client;
+    const { is_mobile } = ui;
+
+    const canResetBalance = () => {
+        return loginid && (accounts[loginid]?.balance || 0) !== 10000;
+    };
```

Comment:
```suggestion
    const can_reset_balance = loginid && (accounts[loginid]?.balance || 0) !== 10000;
```
@hamid-deriv Can be a derived state.